downstream/modules/automation-hub/proc-configure-ansible-galaxy-cli-verify.adoc

```diff
@@ -0,0 +1,52 @@
+[id="proc-configure-ansible-galaxy-cli-verify"]
+
+= Configuring Ansible-Galaxy CLI to verify collections
+
+You can configure Ansible-Galaxy CLI to verify collections. This assures that collections you download are approved by your organization and have not been changed after it was uploaded to {HubName}.
+
+If a collection has been signed by the Galaxy server, the server provides ASCII armored, GPG-detached signatures to verify the authenticity of the MANIFEST.json before using it to verify the collection’s contents. You must opt into signature verification by link:https://docs.ansible.com/ansible/devel/reference_appendices/config.html#galaxy-gpg-keyring[configuring a keyring] for `ansible-galaxy` or providing the path with the `--keyring` option.
+
+.Prerequisites
+
+* Signed collections are available in {HubName} to verify signature.
+* Certified collections can be signed by approved roles within your organization.
+* Public key for verification has been added to the local system keyring.
+
+.Procedure
+
+. To import a public key into a non-default keyring for use with `ansible-galaxy`, run the following command.
++
+----
+gpg --import --no-default-keyring --keyring ~/.ansible/pubring.kbx my-public-key.asc
+----
++
+[NOTE]
+====
+In addition to any signatures provided by the Galaxy server, signature sources can also be provided in the requirements file and on the command line. Signature sources should be URIs.
+====
++
+. Use the `--signature` option to verify the collection name provided on the CLI with an additional signature.
++
+----
+ansible-galaxy collection install namespace.collection
+--signature https://examplehost.com/detached_signature.asc
+--signature file:///path/to/local/detached_signature.asc --keyring ~/.ansible/pubring.kbx
+----
+You can use this option multiple times to provide multiple signatures.
+. Confirm that the collections in a requirements file lists any additional signature sources following the collection’s signatures key, as in the following example.
++
+[source,yaml]
+----
+# requirements.yml
+collections:
+  - name: ns.coll
+    version: 1.0.0
+    signatures:
+      - https://examplehost.com/detached_signature.asc
+      - file:///path/to/local/detached_signature.asc
+
+ansible-galaxy collection verify -r requirements.yml --keyring ~/.ansible/pubring.kbx
+----
++
+When you install a collection from a Galaxy server, the signatures provided by the server are saved along with the installed collections to verify the collection’s authenticity.
```

Comment:
Let's either add "When you install a collection from _Automation Hub_ or Ansible Galaxy" **OR** change Galaxy to Automation Hub.

Reply:
As a matter of fact, I'd like to globally change all mentions of Galaxy to Automation Hub.

Reply:
To clarify further, let's say, 

"when you install a collection from Automation Hub..."

OR 

"when you install a collection from Console.redhat.com Automation Hub or Private Automation Hub..."

Reply:
Okay. Got it. I made the updates globally. Can you take another look in the **Files changed** tab to see if this is what we're looking for. 

Thanks for the feedback thus far! Let me know when you're ready to tackle the introduction for the document, **Managing Red Hat Certified and Ansible Galaxy collections in automation hub**.